src/harness/scripts/harness-airgap-images.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+images="harness-airgapped-images.tar.gz"
+list="airgapped-images.txt"
```

Comment:
We shouldn't be using image list file for push and tag. We can extract image name and tag from 


docker_images=$(docker images --format "{{.Repository}}:{{.Tag}}")

for image in ${docker_images}; do
  image_name="${image%%:*}"
  image_name="${image_name##*/}"

  tag="${image##*:}"

  new_image="${private_registry}/${image_name}:${tag}"

  docker tag "${image}" "${new_image}"

  docker push "${new_image}"
done

Reply:
This wouldn't work since this picks up all the images present in the users docker repo.

Reply:
Added an alternative approach which only picks up images in the harness-airgapped-images.tar.gz